src/test/scala/edu/ie3/simona/agent/grid/DBFSAlgorithmSupGridSpec.scala

```diff
@@ -62,30 +64,35 @@ class DBFSAlgorithmSupGridSpec
     with ImplicitSender
     with DbfsTestGrid {
 
+  private val scheduler: TestProbe = TestProbe("scheduler")
+  private val primaryService: TestProbe = TestProbe("primaryService")
+  private val weatherService: TestProbe = TestProbe("weatherService")
+  private val hsActor: TestProbe = TestProbe("hsActor")
```

Comment:
You probably use `hs` as an abbreviation for Hochspannung? In this case please use `hv` for high voltage instead, please